docs/source/index.rst

```diff
@@ -1,21 +1,46 @@
-.. aatoolbox documentation master file, created by
-   sphinx-quickstart on Fri Oct  1 10:01:58 2021.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+Welcome to AA Toolbox
+=====================
 
-Welcome to aatoolbox's documentation!
-=====================================
+Welcome to AA Toolbox's documentation. Get set up
+and start downloading data using :doc:`quickstart`.
+The rest of the documentation describes the different data sources and
+functionalities in more detail, with a complete reference in
+the :doc:`api` section.
+
+User Guide
+----------
+
+This section includes background information on the AA Toolbox,
+how to get started,
+and detailed guides on the individual data sources plus any
+additional functionality.
 
 .. toctree::
-   :maxdepth: 6
-   :caption: Contents:
+   :maxdepth: 1
+
+   introduction
+   quickstart
+   datasources/codab
```

Comment:
Good point, I think the only way to do so is to add a separate data sources page, let me know if you think it's okay. The source names don't show up in the side menu though, I'd have to change the overall depth and I'm afraid it would get too messy

Reply:
Yes, looks good! I am thinking if a depth of 2 would still be okay, but lets leave it for now.